applications/chats/admin.py

```diff
@@ -3,5 +3,32 @@
 from .models import Message, Room
 
 
-admin.site.register(Room)
-admin.site.register(Message)
+class MessagesInline(admin.TabularInline):
+    model = Message
+    fields = ('writer', 'content', 'created_at', 'is_read', )
+    readonly_fields = ('writer', 'content', 'created_at', 'is_read', )
+
+    def has_add_permission(self, *args):
```

Comment:
What is the purpose of this function if it always returns False?

Reply:
Revoke the permission to add messages from this view